### PR TITLE
3 add touch screen driver

### LINKED ISSUE #3: Add touch screen driver

### PR DESCRIPTION
**Pull request by Sokario**

### CODE REVIEW

**Comment by Sokario** [comment]
@bperraudin This PR needs #2 to be merged, then rebase for it to be reviewed

**Comment by bperraudin** [comment]
This PR should only contain changes related to the issue #3.
It's a problem to list also changes from another branch.
Please remove all necessary commits from this branch.
And finally, please do not ask for review for a PR in draft, except if you need help on how to resolve the issue.

File: boards/shields/zest_display_lcd/zest_display_lcd.overlay [hunk 1]
```diff
@@ -40,3 +41,23 @@
 		};
 	};
 };
+
+&timers1 {
+	status = "okay";
+	st,prescaler = <1000>;
+};
+
```
Comment (clementlgl) [inline boards/shields/zest_display_lcd/zest_display_lcd.overlay:49]: I think we should remove these lines. Prescaler support has been added to the Zest_Core

**Review by clementlgl — CHANGES_REQUESTED** [review]

File: boards/shields/zest_display_lcd/zest_display_lcd.overlay (hunk 1 above)
Comment (Sokario) [reply boards/shields/zest_display_lcd/zest_display_lcd.overlay:49]: It will be done in #5.

**Review by clementlgl — APPROVED** [review]

**Pull request merged**